src/sentry/workflow_engine/endpoints/validators/base/data_condition.py

```diff
@@ -26,7 +26,7 @@ class AbstractDataConditionValidator(
     type = serializers.ChoiceField(choices=[(t.value, t.value) for t in Condition])
     comparison = serializers.JSONField(required=True)
     condition_result = serializers.JSONField(required=True)
-    condition_group_id = serializers.IntegerField(required=True)
+    condition_group_id = serializers.IntegerField(required=False)
```

Comment:
i suppose the `DataCondition` will error if this isn't set and we try to create the object, since `condition_group_id` is required. do we make it optional because we only create these inside of `DataConditionGroupValidator`?

Reply:
Yes - that way the API doesn't need to set the condition_group_id on every request -- also it won't have the DCG id when we initially make a workflow - so if we re-use those validators it'll fail there as well.